lib/driver.js

```diff
@@ -1334,6 +1338,20 @@ class XCUITestDriver extends BaseDriver {
     await this.updateSettings({reduceMotion: isEnabled});
   }
 
+  /**
+   * Set reduceTransparency as 'isEnabled' only when the capabilities has 'reduceTransparency'
+   * The call is ignored for real devices.
+   * @param {?boolean} isEnabled Wether enable reduceTransparency
+   */
+  async setReduceTransparency (isEnabled) {
```

Comment:
I don't think we need a separate driver method for that. Simply call `await this.opts.device.setReduceTransparency(this.opts.reduceTransparency);` in the condition above

Reply:
fixed https://github.com/appium/appium-xcuitest-driver/pull/1464/commits/86120049dc69b3d7669c8bdfacf8c55b86464481